lib/resultsCapturer.js

```diff
@@ -58,15 +58,13 @@ module.exports = function () {
   });
 
   this.registerHandler('StepResult', function (event, callback) {
```

Comment:
So what would you like feature to be set to? 

  this.registerHandler('BeforeFeature', function (event, callback) {
    feature = event;
    callback();
  });

if that object gets renamed to 'feature' then the local 'feature' object will need to be renamed.